br: update PITR FAQ for v7.1 branch (#16090)
pingcap/docs: faq/backup-and-restore-faq.md

### Why is the acceleration of adding indexes feature incompatible with PITR?

> Issue: [#38045](https://github.com/pingcap/tidb/issues/38045)(fixed in v7.0.0)

Reviewer: ```suggestion
Issue: [#38045](https://github.com/pingcap/tidb/issues/38045) (fixed in v7.0.0)
```

> In versions prior to v7.0.0, index data created through the [index acceleration](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) feature cannot be backed up by PITR.

Reviewer: ```suggestion
For TiDB versions earlier than v7.0.0, index data created through the [index acceleration](/system-variables.md#tidb_ddl_enable_fast_reorg-new-in-v630) feature cannot be backed up by PITR.
```

### What should I do if the error `execute over region id` is returned when I perform PITR?

> Issue: [#37207](https://github.com/pingcap/tidb/issues/37207)(fixed in v6.6.0)

Reviewer: ```suggestion
Issue: [#37207](https://github.com/pingcap/tidb/issues/37207) (fixed in v6.6.0)
```

> Issue: [#37207](https://github.com/pingcap/tidb/issues/37207)(fixed in v6.6.0 )

Reviewer: ```suggestion
Issue: [#37207](https://github.com/pingcap/tidb/issues/37207)(fixed in v6.6.0)
```

> In versions prior to v6.6.0, the error of `execute over region id` may be displayed when PITR restoration is performed. This issue usually occurs when you enable log backup during a full data import and afterward perform a PITR to restore data at a time point during the data import.

Reviewer: ```suggestion
For TiDB versions earlier than v6.6.0, you might encounter the error of `execute over region id` during PITR recovery. This issue usually occurs when you enable log backup during a full data import and afterward perform a PITR to restore data at a time point during the data import.
```

Specifically, there is a probability that this issue occurs if there are a large number of hotspot writes for a long time (such as 24 hours) and if the OPS of each TiKV node is larger than 50k/s (you can view the metrics in Grafana: **TiKV-Details** -> **Backup Log** -> **Handle Event Rate**).

> For versions prior to v6.6.0, it is recommended that you perform a snapshot backup after the data import and perform PITR based on this snapshot backup.

Reviewer: ```suggestion
For TiDB versions earlier than v6.6.0, it is recommended that you perform a snapshot backup after the data import and perform PITR based on this snapshot backup.
```
